ur_robot_driver/resources/ursim_driver/ursim/aux/ros_control.urscript

```diff
@@ -0,0 +1,143 @@
+{{BEGIN_REPLACE}}
```

Comment:
Why is this file needed. It should use the one from the client_library!

Reply:
agreed